### PR TITLE
Added a general product listing page, will provide with specific products soon.

### PR DESCRIPTION
**Pull request by j24m**
Added a general product listing page, will provide with specific products soon.

### CODE REVIEW

File: productpage.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+.main-product-container {
+    overflow: hidden;
+    width: 90%;
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin: auto;
+    box-shadow: 2px 2px 10px black;
```
Comment (zoef-yt) [inline productpage.css]: Use var for colour :)

File: productpage.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+.click-image{
+    position:absolute;
+    bottom:1rem;
+    left:1rem;
+    color:var(--footer-icon-hover-color)
+}
+
+.footer {
+
```
Comment (zoef-yt) [inline productpage.css:78]: remove extra spacing

**Review by zoef-yt — APPROVED** [review]
Good Class names. Good code. Easily understandable. Good to merge with the changes.

**Review by samyakshah3008 — APPROVED** [review]
Looks great to merge!

**Review by mehulsatardekar — APPROVED** [review]
You can merge this branch didn't find any issues. 🚀

File: productpage.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+.title-container {
+    margin: 3rem;
+    font-size: 3rem;
+}
+
+.product-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```
Comment (rahulyadav139) [inline productpage.css:26]: for the same column width value, you can use the repeat function.

**Comment by bhatvikrant** [comment]
Its already merged, but the code looks good to me @j24m 👌🏻

**Pull request merged**